### DISCUSSION TITLE
n-panel + PBR

### DISCUSSION THREAD

**Discussion by shaqualeek** [carson-katri/dream-textures#93] [Ideas]
Hi. I think this has great potential and I would like to thank you for your effort in the first place and for making it free for the community.

1/ I suggest making the addon part of the n-panel so that it is not necessary to open it again every time.

2/ This is probably not so easy task to ask for, but It would be great to implement a feature for textures to also generate height/normal/roughness/metallic passes. Might be fake..just to speed up routine steps after generating some interesting albedo iteration. Something like Materialize does for example. // https://boundingboxsoftware.com/materialize/

**Comment by carson-katri** [comment]
Check out #39 which moves the UI to the side panel. This will be included in the next release.

As for PBR textures, I will be looking into what options are already available (and possible to integrate)

**Reply by mangoplane** [reply]
Hi Carson, 

I'm just wondering how the PBR texture feature is coming along. I have a large repository of PBR textures, which I found online, that might be of use. Do you see this being realised soon, if it isn't already present?

Thanks.
